OpenSim/Simulation/Model/ContactMesh.cpp

```diff
@@ -115,7 +118,7 @@ SimTK::ContactGeometry::TriangleMesh* ContactMesh::
     file.close();
     mesh.loadFile(filename);
     if (restoreDirectory) IO::chDir(savedCwd);
-        
+    _decorativeGeometry.reset(new SimTK::DecorativeMesh(mesh));
     return new SimTK::ContactGeometry::TriangleMesh(mesh);
```

Comment:
This is a little weird to me; the two pointers are handled differently. I would prefer if `loadMesh()` were changed so its return type is null, and that the last line is changed to:

```
_geometry.reset(new SimTK::ContactGeometry::TriangleMesh(mesh));
```

And that the doxygen for this method is updated accordingly.